examples/step-9/step-9.cc

```diff
@@ -381,33 +215,131 @@ namespace Step9
     (void)component;
     Assert(component == 0, ExcIndexRange(component, 0, 1));
 
-    const double sine_term =
-      std::sin(16 * numbers::PI * std::sqrt(p.norm_square()));
-    const double weight = std::exp(-5 * p.norm_square()) / std::exp(-5.);
+    const double sine_term = std::sin(16. * numbers::PI * p.norm());
+    const double weight    = std::exp(-5. * p.norm_square()) / std::exp(-5.);
```

Comment:
I would rather change this than the formulation in the introduction. Dividing by `std::exp(-5.)` surely is more expensive than multiplying in the exponent.